### DISCUSSION TITLE
How to perform signaling pathway and Gene ontology analysis using R in scRNA-seq data?

### DISCUSSION THREAD

**Discussion by Amin2130538** [satijalab/seurat#6080] [General]
Currently, I am performing scRNA-seq data analysis using R for WT and mutant mice model data. I have already done integration and clustering with manual annotation based on the marker genes. Now, I would like to perform signaling pathway and gene ontology analysis of my scRNA-seq data. 

For example, in the previous paper they performed the gene ontology analysis in their scRNA-seq data. Example figure:
![Screen Shot 2022-06-15 at 14 00 32](https://user-images.githubusercontent.com/96256252/173740920-ecea9c4e-0374-4012-852f-5e2c3b3a373b.png)

 
So, How to I can perform these kind of analysis based on my scRNA-seq data in R? Your suggestion, code example or weblink are highly appreciated. Thank you.....

**Comment by samuel-marsh** [comment]
Hi,

Going to move this post to the discussion section as it doesn't pertain to Seurat directly but more scRNA-seq analysis as a whole.  I would also recommend posting to bioinformatics focused forum (e.g., Biostars as you may get faster more informative responses there).  To recapitulate what was done in the figure you posted above I suggest checking the methods the authors used in the paper's methods section and/or contacting them specifically to see if they might be willing to share code if it's not posted publicly.

Best,
Sam
